docs/api.md

```diff
@@ -6,7 +6,15 @@ title: API
 Call `katex.render` with a TeX expression and a DOM element to render into:
 
 ```js
-katex.render("c = \\pm\\sqrt{a^2 + b^2}", element);
+try {
+    katex.render("c = \\pm\\sqrt{a^2 + b^2}", element);
+} catch (e) {
+    if (e instanceof katex.ParseError) {
+        // KaTeX can't parse the expression
+    } else {
+        throw e;
+    }
+}
```

Comment:
I'd personally rather have this more complex example in the "Handling Errors" section.  We could either return this example as it was, or e.g. encourage the use of `{throwOnError: false}`, which should cover the most common use case -- and then refer to Handling Errors for alternatives.  What do you think?